testprojects/pants-plugins/3rdparty/pants/BUILD

```diff
@@ -0,0 +1 @@
+pants_requirement()
```

Comment:
Should this... have an argument?

Reply:
Not necessarily, and not needed for this purpose. See [`test_pants_requirement.py`](https://github.com/cosmicexplorer/pants/blob/a1e30ff19aa8fd54d994fcb2178895513cbce4c4/tests/python/pants_test/backend/python/test_pants_requirement.py) for an example of providing a name and not.